app/channels/watcher_channel.rb

```diff
@@ -0,0 +1,12 @@
+class WatcherChannel < ApplicationCable::Channel
+  CHANNEL_NAME = "WatcherChannel"
+  def subscribed
+    stream_from "watcher_channel#{params[:user_id]}"
```

Comment:
aici doar de amorul artei hai sa facem
```
if user = User.find params.require(:user_id)
  stream_from "watcher_channel#{user.id}"
end
```

ca sa fim siguri ca idul e valid (si in viitor sa tinem minte sa inlocuim cu token)